nbgrader/docs/source/user_guide/tests.yml

```diff
@@ -0,0 +1,310 @@
+#kernel_name:
```

Comment:
This file is not getting copied when running `nbgrader quickstart` (which copies the example assignments from the docs), meaning that running `nbgrader generate_assignment` doesn't work for notebooks that include auto tests. Can you update the quickstart code so that it is? You can test that it works by running the following:

```
cd /tmp
nbgrader quickstart course101
cd course101
nbgrader generate_assignment ps2
```

Reply:
We updated the `quickstartapp.py` to copy the `tests.yml` file as well

`
# copying the tests.yml file to the course directory
tests_file_path = os.path.abspath(os.path.join(
    os.path.dirname(__file__), '..', 'docs', 'source', 'user_guide', 'tests.yml'))
shutil.copyfile(tests_file_path, course_path)
`